extensions/shared/register-jetpack-block.js

```diff
@@ -12,9 +13,21 @@ import getJetpackExtensionAvailability from './get-jetpack-extension-availabilit
 import withHasWarningIsInteractiveClassNames from './with-has-warning-is-interactive-class-names';
 import wrapPaidBlock from './wrap-paid-block';
 
+const availableBlockTags = {
+	paid: __( 'paid', 'jetpack' ),
+	beta: __( 'beta', 'jetpack' ),
```

Comment:
Initially, we didn't translate this since it was just internal for a8c only, but not a biggie to translate it either.

Reply:
I've left this translatable for now, but I'm happy to remove it from translation if you'd prefer!